src/mixins/name-request-mixin.ts

```diff
@@ -61,6 +61,7 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
       nr.state &&
       nr.expirationDate &&
       nr.legalType &&
+      (nr.request_action_cd === NameRequestTypes.NEW) &&
```

Comment:
Added this check for valid NR action code per discussion with Vysakh and Cameron.